activerecord/lib/active_record/connection_adapters/postgresql/oid/bit.rb

```diff
@@ -30,7 +30,7 @@ def initialize(value)
             end
 
             def to_s
-              value
+              value.is_a?(::String) ? value : value.to_s
```

Comment:
I guess we could use just `value.to_s` on this line, since `String#to_s` returns `self`